### PR TITLE
Miscellaneous native text styling fixes

### PR DESCRIPTION
**Pull request by chromakode**
:eyeglasses: @keybase/react-hackers

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 7666485c927a...268a26d57368 introduce visual changes on linux. <details><summary>:mag_right: 7 new</summary>

 * Added: **Text-Body_Wrap_Terminal**  
   Text-Body_Wrap_Terminal rendered
 * Added: **Text-Body_Wrap_Success**  
   Text-Body_Wrap_Success rendered
 * Added: **Text-Body_Wrap_Normal**  
   Text-Body_Wrap_Normal rendered
 * Added: **Text-Body_Wrap_Information**  
   Text-Body_Wrap_Information rendered
 * Added: **Text-Body_Wrap_High_Risk**  
   Text-Body_Wrap_High_Risk rendered
 * Added: **Text-Body_Wrap_Documentation**  
   Text-Body_Wrap_Documentation rendered
 * Added: **Text-Body_Wrap_Announcements**  
   Text-Body_Wrap_Announcements rendered
</summary>

File: shared/chat/conversations-list/index.native.js [hunk 1]
```diff
@@ -61,8 +61,8 @@ const Avatars = ({participants, youNeedToRekey, participantNeedToRekey, isMuted,
 const TopLine = ({hasUnread, showBold, participants, subColor, timestamp, usernameColor}) => {
   const boldOverride = showBold ? globalStyles.fontBold : null
   return (
-    <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', maxHeight: 17, minHeight: 17}}>
-      <Box style={{...globalStyles.flexBoxRow, flex: 1, height: 17, position: 'relative'}}>
+    <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', maxHeight: 18, minHeight: 18}}>
```
Comment (chrisnojima) [inline shared/chat/conversations-list/index.native.js]: i'm working on changing the styling of this whole component now so lets remove this

File: shared/common-adapters/markdown.native.js [hunk 2]
```diff
@@ -47,13 +47,13 @@ function previewCreateComponent (style) {
   return function (type, key, children, options) {
     switch (type) {
       case 'markup':
-        return <Text type='Body' key={key} lineClamp={1}>{children}</Text>
+        return <Text type='Body' key={key} style={style} lineClamp={1}>{children}</Text>
```
Comment (chrisnojima) [inline shared/common-adapters/markdown.native.js]: i have this fix in my stuff also

**Comment by keybase-ci-visdiff** [comment]
The commits 7666485c927a...0745ee2aa55d introduce visual changes on linux. <details><summary>:mag_right: 7 new</summary>

 * Added: **Text-Body_Wrap_Terminal**  
   Text-Body_Wrap_Terminal rendered
 * Added: **Text-Body_Wrap_Success**  
   Text-Body_Wrap_Success rendered
 * Added: **Text-Body_Wrap_Normal**  
   Text-Body_Wrap_Normal rendered
 * Added: **Text-Body_Wrap_Information**  
   Text-Body_Wrap_Information rendered
 * Added: **Text-Body_Wrap_High_Risk**  
   Text-Body_Wrap_High_Risk rendered
 * Added: **Text-Body_Wrap_Documentation**  
   Text-Body_Wrap_Documentation rendered
 * Added: **Text-Body_Wrap_Announcements**  
   Text-Body_Wrap_Announcements rendered
</summary>

**Comment by chrisnojima** [comment]
@chromakode this still includes changes to conversation-list/index.native.js

**Comment by chrisnojima** [comment]
we need that inner box to make ellipsis work correctly

**Comment by chrisnojima** [comment]
i'll try and fix this in my inbox restyling branch

**Pull request merged**